interface.js

```diff
@@ -0,0 +1,129 @@
+signInSheet()
+signUpSheet()
+
+function signInSheet() {
+  $('.sign-in').click(function() {
```

Comment:
You have a `sign-in` class on two elements in your html, which means when clicking on the homepage title, it takes the user to the sign-in. Would it be worth updating these to have two separate ids rather than a class? then you can focus your sign in click on the sign in button

This isn't related to your code, but thought you might find it useful for future projects - on the weekend i came across something called _event bubbling_, which is helpful when you have lots of divs inside other divs and want to create click events on the parent div.